x-pack/plugins/transform/public/app/sections/transform_management/components/edit_transform_flyout/edit_transform_flyout_callout.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+
+import { i18n } from '@kbn/i18n';
+
+import {
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiLink,
+  EuiTextColor,
+} from '@elastic/eui';
+
+import { useDocumentationLinks } from '../../../../hooks/use_documentation_links';
+export const EditTransformFlyoutCallout: FC = () => {
+  const { esTransformUpdate } = useDocumentationLinks();
+
+  return (
+    <EuiCallOut>
+      <EuiFlexGroup>
+        <EuiFlexItem grow={false}>
+          <EuiIcon type="help" />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          {' '}
+          <EuiTextColor color="subdued">
+            {i18n.translate('xpack.transform.transformList.editFlyoutCalloutText', {
+              defaultMessage:
+                'This form allows you to update an existing transform. The list of properties that you can update is a subset of the list that you can define when you create a transform.',
```

Comment:
I'd remove the word `existing` as 'update' implies it exists.

Reply:
FYI The text is a variation of what we have in the docs with the same wording https://www.elastic.co/guide/en/elasticsearch/reference/master/update-transform.html#update-transform-desc — Maybe we need to decide: Do we want to copy any snippets from there at all or just link to the docs without additional text? (Suggest to change if necessary in follow-up).